src/lib/components/schedule/schedule-form-view.svelte

```diff
@@ -46,8 +42,8 @@
     schedule?: Schedule,
   ) => void;
 
-  let namespace = $page.params.namespace;
-  let scheduleId = $page.params.schedule;
+  $: namespace = $page.params.namespace;
+  $: scheduleId = $page.params.schedule;
```

Comment:
These can't be reactive. If you try to click on Back to Schedules, you will see it returns you to `/namespaces/undefined/schedules`

Reply:
Otherwise you can update backHref to be reactive. $: backHref = ...

Reply:
Actually it needs to be let namespace, let scheduleId because we use those values below as well for the form. We could update those to be reactive too but for now its best to just make these lets

Reply:
Sorry, changed in two places, missed one. Thanks for catching.